doc/api.md

```diff
@@ -203,25 +205,27 @@ Note that there is a 1:1 correspondence between payload fields and DB fields (bu
-| `static_attributes`       | `staticAttributes`   | List of static attributes to append to the entity. All the updateContext requests to the CB will have this set of attributes appended.                                                                                                                               | `[ { "name": "attr_name", "type": "Text" } ]` |
+| Payload Field             | DB Field             | Definition                                                                                                                                                                       | Example of value                              |
+| ------------------------- | -------------------- | :------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | :-------------------------------------------- |
+| `device_id`               | `id`                 | Device ID that will be used to identify the device.                                                                                                                              | UO834IO                                       |
+| `service`                 | `service`            | Name of the service the device belongs to (will be used in the fiware-service header).                                                                                           | smartGondor                                   |
+| `service_path`            | `subservice`         | Name of the subservice the device belongs to (used in the fiware-servicepath header).                                                                                            | /gardens                                      |
+| `entity_name`             | `name`               | Name of the entity representing the device in the Context Broker                                                                                                                 | ParkLamplight12                               |
+| `entity_type`             | `type`               | Type of the entity in the Context Broker                                                                                                                                         | Lamplights                                    |
+| `timezone`                | `timezone`           | Time zone of the sensor if it has any                                                                                                                                            | America/Santiago                              |
+| `timestamp`               | `timestamp`          | Optional flag about whether or not to addthe TimeInstant attribute to the device entity created, as well as a TimeInstant metadata to each attribute, with the current timestamp | true                                          |
```

Comment:
Diff shows:

![imagen](https://user-images.githubusercontent.com/1534240/87909731-13c0ca00-ca69-11ea-8fdb-da6d41adc01f.png)

Is the removal of the sentence about NGSI-LD ("With NGSI-LD ...") correct?

Reply:
Fixed  ddb8cfe